docs/document/website/versioned_docs/version-1.4.0/guides/version-conflicts-issue-solution.md

```diff
@@ -55,8 +55,49 @@ document.body.appendChild(button);
 Users using Kintone UI Component through the CDN will always have the latest version.<br>
 Therefore, after the release of v1.4.0, some of your apps and plug-ins will not work correctly. <br>
 
-As explained in the [Using UMD](#using-umd) section, you now need to change from `Kuc` to `Kucs['1.x.x']`.
+As explained in the [Using UMD](#using-umd) section, you may still use `Kuc` as a global variable but be aware of the possible version conflicts errors that may occur.
 
 ### Using npm
 
 Users using the Kintone UI Component package through npm do not need to take any action but note the changes in how the components' tags and class names render. See the [Solution](#solution) section.
+
+## Caution required
+
+### For users using versions before v1.4.0
+
+Below are the common version conflicts errors that might occur:
+#### 1. When importing multiple KUC packages and files (both ESM and UMD) of the same version or different versions:
+
+In the example below, we load a v1.2.0 `kuc.min.js` file in the Kintone system and a v1.3.0 `kuc.min.js` file in our app. An `Illegal constructor` error will show when we try to create a new KUC Button component.
+
+![Illegal constructor error when importing multiple kuc.min.js files](assets/UMD_multi_files.jpeg)
+
+#### 2. When importing multiple KUC packages (ESM) and using custom HTML tags directly:
+
+For example, we have two scripts as follow:
+- One script uses v1.2.0 and specifies the text color of Dropdown to be green.
+- One script uses v1.3.1 and specifies the text color of Dropdown to be red
+
+In the first case, we have the v1.2.0 script loaded first and v1.3.1 loaded last. We can see that the text color of Dropdown in the v1.3.1 script is overridden by v1.2.0 script.
```

Comment:
How about unifying the version to use with 1. case? (v1.2.0 and v1.3.0)
About the image, you can override only the text area!